features/all-positions/AllSponsors.tsx

```diff
@@ -0,0 +1,250 @@
+import {
```

Comment:
splitting these is good. will fit well in the other PR with click to liquidate functions.